src/main/java/com/icthh/xm/uaa/web/rest/vm/AuthorizeUserVm.java

```diff
@@ -0,0 +1,24 @@
+package com.icthh.xm.uaa.web.rest.vm;
+
+import com.icthh.xm.uaa.service.dto.UserDTO;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+
+public class AuthorizeUserVm extends UserDTO {
+
+    @NotEmpty
+    @Getter
+    private String login;
+
+    @Setter
+    @Getter
+    private String captcha;
+
+    public AuthorizeUserVm(String login, String captcha) {
```

Comment:
check is it work by http (jackson need additional annotations)

Reply:
it worked fine during local testing
i will update this request to avoid depending on UserDTO

Reply:
done